jetty-client/src/main/java/org/eclipse/jetty/client/http/HttpChannelOverHTTP.java

```diff
@@ -80,7 +83,10 @@ public void send()
     {
         HttpExchange exchange = getHttpExchange();
         if (exchange != null)
-            sender.send(exchange);
+        {
+            sender.send( exchange );
+            _outMessages.add( 1 );
```

Comment:
inconsistent formatting, sometimes you have `(1)` and sometimes `( 1 )`.  Also I think there is an increment method?